Apps/W1/Shopify/test/Metafields/ShpfyCompanyMetafieldsTest.Codeunit.al

```diff
@@ -0,0 +1,191 @@
+codeunit 139617 "Shpfy Company Metafields Test"
```

Comment:
what about some tests for the export of metafields?

Reply:
tried to cover only my changes, and for the export it uses the same code, only different value is returned here (general Metafields functionality)

![image](https://github.com/user-attachments/assets/dad9503d-ddff-44a9-bfa9-cb3c37529fc9)

I could write test for expected value from GetOwnerTypeName or maybe its worth creating code to generally test the metafield export?

Reply:
We'll have to do a more general coverage anyway (as part of the "cover old features with tests).

But regardless of that, think of it this way, moving forward, we want to know that when exporting customers and companies, metafields are exported as well. So at the very least, I'd want a test, that will check if the "UpdateMetafields" is executed. Sorta of like having a tests for the "acceptance criteria". If the requirement is that exporting entities exports metafields, than having tests that "ensure" that makes sure we don't break it in the future.